doc/cephadm/osd.rst

```diff
@@ -202,77 +220,105 @@ When no PGs are left on the OSD, it will be decommissioned and removed from the
+instead assigned a 'destroyed' flag.
 
 **Preserving the OSD ID**
 
-The previously-set 'destroyed' flag is used to determine OSD ids that will be reused in the next OSD deployment.
+The 'destroyed' flag is used to determine which OSD ids will be reused in the
+next OSD deployment.
 
-If you use OSDSpecs for OSD deployment, your newly added disks will be assigned the OSD ids of their replaced
-counterparts, assuming the new disks still match the OSDSpecs.
+If you use OSDSpecs for OSD deployment, your newly added disks will be assigned
+the OSD ids of their replaced counterparts. This assumes that the new disks
+still match the OSDSpecs.
 
-For assistance in this process you can use the '--dry-run' feature.
+Use the ``--dry-run`` flag to make certain that the ``ceph orch apply osd`` 
+command does what you want it to. The ``--dry-run`` flag shows you what the
+outcome of the command will be without making the changes you specify. When
+you are satisfied that the command will do what you want, run the command
+without the ``--dry-run`` flag.
 
-Tip: The name of your OSDSpec can be retrieved from **ceph orch ls**
+.. tip::
 
-Alternatively, you can use your OSDSpec file::
+  The name of your OSDSpec can be retrieved with the command ``ceph orch ls``
 
-    ceph orch apply osd -i <osd_spec_file> --dry-run
-    NAME                  HOST  DATA     DB WAL
-    <name_of_osd_spec>    node1 /dev/vdb -  -
+Alternatively, you can use your OSDSpec file:
 
+.. prompt:: bash #
 
-If this matches your anticipated behavior, just omit the --dry-run flag to execute the deployment.
+  ceph orch apply osd -i <osd_spec_file> --dry-run
 
+Expected output::
 
-Erase Devices (Zap Devices)
----------------------------
+  NAME                  HOST  DATA     DB WAL
+  <name_of_osd_spec>    node1 /dev/vdb -  -
 
-Erase (zap) a device so that it can be reused. ``zap`` calls ``ceph-volume zap`` on the remote host.
 
-::
+When this output reflects your intention, omit the ``--dry-run`` flag to
+execute the deployment.
+
 
-     orch device zap <hostname> <path>
+Erasing Devices (Zapping Devices)
+---------------------------------
 
-Example command::
+Erase (zap) a device so that it can be reused. ``zap`` calls ``ceph-volume
+zap`` on the remote host.
+
+.. prompt:: bash #
+
+  orch device zap <hostname> <path>
+
+Example command:
+
+.. prompt:: bash #
 
-     ceph orch device zap my_hostname /dev/sdx
+  ceph orch device zap my_hostname /dev/sdx
 
 .. note::
-    Cephadm orchestrator will automatically deploy drives that match the DriveGroup in your OSDSpec if the unmanaged flag is unset.
-    For example, if you use the ``all-available-devices`` option when creating OSDs, when you ``zap`` a device the cephadm orchestrator will automatically create a new OSD in the device .
+    If the unmanaged flag is unset, cephadm orchestrator automatically deploys drives that match the DriveGroup in your OSDSpec. 
```

Comment:
```suggestion
    If the unmanaged flag is unset, cephadm automatically deploys drives that match the DriveGroup in your OSDSpec. 
```

Reply:
I've removed "orchestrator" from this sentence.